plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Progress.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.triplet.gradle.play.internal
+
+import com.google.api.client.googleapis.media.MediaHttpUploader
+import com.google.api.services.androidpublisher.AndroidPublisherRequest
+import org.gradle.internal.logging.progress.ProgressLogger
+import kotlin.math.roundToInt
+
+internal fun <T> AndroidPublisherRequest<T>.trackProgress(
```

Comment:
This is really nice. Let's call it trackUploadProgress as there are some very phrases in here that are very specific to artifact uploads

Reply:
Can do!